Source/ElasticLINQ.Test/Test/TestableElasticQueryTests.cs

```diff
@@ -52,7 +52,7 @@ public static void ExplicitGetEnumeratorCapturesQueryOnContext()
 
             var dummy = ((IEnumerable) new TestableElasticQuery<FakeClass>(context).Where(f => f.Name == "a")).GetEnumerator();
 
-            Assert.Equal(1, context.Requests.Count);
+            Assert.Single(context.Requests);
```

Comment:
Is this general clean up or were the tests failing with the old logic?

Reply:
Unintended cleanup. I used a lot of my resharper binds to delete the relink the csprojs and in doing so some of these got changed. I can totally revert these if you want to reduce the noise.